### ISSUE TITLE
Please make React a peerDependency instead of dependency

### ISSUE THREAD

**Issue by pcaplan-teladoc** [andrewhathaway/Winterfell#99]
Having react as a dependency is causing errors due to multiple copies of react being loaded. To address this, please move react and react-dom from dependencies to peerDependencies. 

See https://stackoverflow.com/questions/28519287/what-does-only-a-reactowner-can-have-refs-mean

**Comment by cschram** [comment]
React and React DOM are still listed as devDependencies, which is causing errors mentioned above with conflicting versions being loaded. Specifically I'm getting this error:

```
Uncaught TypeError: this.updater.enqueueCallback is not a function
    at RadioOptionsInput../node_modules/winterfell/node_modules/react/lib/ReactBaseClasses.js.ReactComponent.setState (ReactBaseClasses.js:64)
    at RadioOptionsInput.handleChange (radioOptionsInput.js:29)
    at HTMLUnknownElement.callCallback (react-dom.development.js:542)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:581)
    at Object.invokeGuardedCallback (react-dom.development.js:438)
    at Object.invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:452)
    at executeDispatch (react-dom.development.js:836)
    at executeDispatchesInOrder (react-dom.development.js:858)
    at executeDispatchesAndRelease (react-dom.development.js:956)
    at executeDispatchesAndReleaseTopLevel (react-dom.development.js:967)
    at Array.forEach (<anonymous>)
    at forEachAccumulated (react-dom.development.js:935)
    at processEventQueue (react-dom.development.js:1112)
    at runEventQueueInBatch (react-dom.development.js:3607)
    at handleTopLevel (react-dom.development.js:3616)
    at handleTopLevelImpl (react-dom.development.js:3347)
    at batchedUpdates (react-dom.development.js:11082)
    at batchedUpdates (react-dom.development.js:2330)
    at dispatchEvent (react-dom.development.js:3421)
```

This is caused by my app using React DOM 16.2, but Winterfell is using React 15.6 listed in it's devDependencies.

**Comment by andrewhathaway** [comment]
Fixed by #103!
